runtime/service/src/test/java/org/apache/polaris/service/catalog/iceberg/AbstractIcebergCatalogTest.java

```diff
@@ -2283,6 +2287,96 @@ public void testTableOperationsDoesNotRefreshAfterCommit(boolean updateMetadataO
     }
   }
 
+  @Test
+  public void testTableInternalPropertiesStoredOnCommit() {
+    Assumptions.assumeTrue(
+        requiresNamespaceCreate(),
+        "Only applicable if namespaces must be created before adding children");
+
+    catalog.createNamespace(NS);
+    catalog.buildTable(TABLE, SCHEMA).create();
+    catalog.loadTable(TABLE).newFastAppend().appendFile(FILE_A).commit();
+    Table afterAppend = catalog.loadTable(TABLE);
+    EntityResult schemaResult =
```

Comment:
```suggestion
    EntityResult namespaceResult =
```
[nit] Let's use `namespace` here to be consistent with polaris' terminology